ls1intum/Artemis
# Bugfix/removed trim from general feedback text (#1465)

### Checklist


### Motivation and Context

Better usability for tutors. 

Fixes #1465 

### Description

Removed the trim from the text 

### Steps for Testing
 
Reproduce steps are also in the issue. 
Steps:   
1. Log in to Artemis
2. As a tutor, open up an assessment.
3. Go to the general feedback.
4. Type in "test t" and hit the backspace button. It should only remove t, and not " t". 

### Screenshots


Fixed: 
![remove-backspace](https://user-images.githubusercontent.com/63906424/82064823-90cc6d80-96cd-11ea-8a1b-ea7283e20206.gif)
